adagrams/game.py

```diff
@@ -1,11 +1,155 @@
+import random
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+
 def draw_letters():
-    pass
+
+    final_string = []
+    letters_frequency = []
+    letters_list = []
+    
+    for letters in LETTER_POOL:
+        letters_frequency.append(LETTER_POOL[letters])
+    
+
+    for letter in LETTER_POOL:
+        letters_list.append(letter)
+
+    while len(final_string) < 10:
+        letter_index = random.randint(0, len(LETTER_POOL) -1)
```

Comment:
For an actually weighted distribution, there should be 98 letters to pull from. However, your code is pulling from 26 letters (all the letters in the alphabet). Right now, the letter `Z` is just as likely to be pulled as letter `E` (though they can only appear up to their respective limits). How could this function be changed to utilize the weighted distribution?